apps/server/src/modules/authorization/authorization.service.ts

```diff
@@ -50,6 +54,12 @@ export class AuthorizationService extends BasePermissionManager {
 	}
 
 	async getUserWithPermissions(userId: EntityId) {
-		return this.loader.getUserWithPermissions(userId);
+		try {
+			const userWithPermissions = await this.loader.getUserWithPermissions(userId);
+
+			return userWithPermissions;
+		} catch (err) {
+			throw new ForbiddenException();
```

Comment:
Shall we really map every possible error to a ForbiddenException? What's when there is just some temporary error in the database?

I have to think about a bug in dPersonalmanagement that annoys me from time to time: I want to login and it says "wrong credentials" though I'm sure they are not wrong because I try several times. And then after some time I can login again with my credentials. That could be caused by similar error handling.

Reply:
let me think about it in context of db connection lost

Reply:
I pass the error in body of the next error

Reply:
What difference does that make to pass the error? 

But anyway: I just talked to Max about it and we also came to the conclusion that it's okay to do it this way here.